### PR TITLE
30veryEasy

### PR DESCRIPTION
**Pull request by Tubka**

### CODE REVIEW

File: very easy/19.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
 function sum(n) {
-	
+  let a = 0;
```
Comment (mras2303) [inline very easy/19.js]: `a` is the name of variable which say nothing to me

File: very easy/24.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
 function isPlural(word) {
-	
+  return [...word][word.length - 1] == "s";
```
Comment (mras2303) [inline very easy/24.js]: I don't understand using that spread operator. Could you explain me that?

File: very easy/29.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
 function calculateExponent(num, exp) {
-	
+  return Math.pow(num, exp);
```
Comment (mras2303) [inline very easy/29.js]: In newer version of ES6, we can use `**` operator to exponent numbers

**Review by hayuna — APPROVED** [review]
I think truly @dam-orzel need to teach you ECMAScript :(

File: very easy/24.js (hunk 2 above)
Comment (Tubka) [reply very easy/24.js]: i thought that the "length" property is only available for spread operator - changed

File: very easy/19.js (hunk 1 above)
Comment (Tubka) [reply very easy/19.js]: name changed

File: very easy/29.js (hunk 3 above)
Comment (Tubka) [reply very easy/29.js]: ok, i changed -> num ** exp

File: very easy/19.js (hunk 1 above)
Comment (hayuna) [inline very easy/19.js]: You can use reduce

File: very easy/24.js (hunk 2 above)
Comment (hayuna) [inline very easy/24.js]: This spread operator is unnecessary

File: very easy/25.js [hunk 4]
```diff
@@ -1,9 +1,9 @@
 function ctoa(c) {
-	
+  return c.charCodeAt(0);
```
Comment (hayuna) [inline very easy/25.js]: 0 is default value in this method. You can skip it

File: very easy/29.js (hunk 3 above)
Comment (hayuna) [inline very easy/29.js]: Good, but in ES7 has added operator of exponent - **

**Comment by hayuna** [comment]
- 19 You can use reduce
- 24 This spread operator is unnecessary
- 25 0 is default value in this method. You can skip it
- 29 Good, but in ES7 has added operator of exponent - `**`

Result: 93.33%

**Pull request closed without merging**